server/sonar-server/src/main/java/org/sonar/server/startup/GeneratePluginIndex.java

```diff
@@ -54,8 +51,8 @@ void writeIndex(File indexFile) throws IOException {
     FileUtils.forceMkdir(indexFile.getParentFile());
     Writer writer = new FileWriter(indexFile, false);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)Remove this use of constructor "FileWriter(File,boolean)"[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1943)